src/coral_models/wav2vec2.py

```diff
@@ -236,7 +274,7 @@ def load_saved(self) -> PreTrainedModelData:
 
         model = Wav2Vec2ForCTC.from_pretrained(self.cfg.hub_id, token=True)
         data_collator = DataCollatorCTCWithPadding(
-            processor=processor, padding="longest"
+            processor=processor, padding="max_length"
```

Comment:
I didn't completely test this, but I think using `longest` caused an issue when running the finetuning with DDP, but I can do some tests to see whether it's necessary. Ideally we shouldn't use max_length, if we can avoid it anyway.

Reply:
Okay, I tested it and it _is_ required in a distributed setting. I've now tried to accomodate for both by making it a config parameter which defaults to "longest", but that it is automatically set to "max_length" in a distributed setting (and this will be logged to maintain transparency).

Reply:
Looks very nice!